earth_enterprise/src/fusion/autoingest/tools/getop.cpp

```diff
@@ -242,13 +243,17 @@ main(int argc, char *argv[])
         }
 
         outline("");
-        outline("Number of cached assets: %u", taskLists.num_assets_cached);
-        outline("Number of cached asset versions: %u",
-                taskLists.num_assetversions_cached);
-        outline("Total memory used by cached assets: %s",
-                readableMemorySize(taskLists.asset_cache_memory).c_str());
-        outline("Total memory used by cached asset versions: %s",
-                readableMemorySize(taskLists.version_cache_memory).c_str());
+        if (MiscConfig::Instance().LimitMemoryUtilization) {
```

Comment:
Adding a comment here to reflect our prior conversation.
As discussed, pending performance testing, it would be nice to have the memory usage calculated and displayed if the performance cost is relatively negligible.